rotkehlchen/globaldb/upgrades/v3_v4.py

```diff
@@ -256,17 +258,69 @@ def _add_optimism_contracts(cursor: 'DBCursor', eth_scan_abi_id: int, multicall_
     log.debug('Exit _add_optimism_contracts')
 
 
+def _copy_assets_from_packaged_db(
+        cursor: 'DBCursor',
+        assets_ids: list[str],
+        root_dir: Path,
+) -> None:
+    """
+    Copy missing assets from the packaged globaldb to the globaldb
```

Comment:
Should this be only in this upgrade? Shouldn't we do this more often also? Kind of a check, if a user has done a booboo and deleted an asset that should be constant and in the DB?

Reply:
in this case it is required. If a user deletes a constant asset is more of his fault I would say. We could do this it nevertheless but I guess someone wouldn't delete an asset if there is no reason for that or by mistake

Reply:
Sure I am just mentioning it so we keep in mind it may make sense to do this also periodically at some point.